packages/camera/camera/ios/Classes/FLTCam_Test.h

```diff
@@ -5,6 +5,17 @@
 #import "FLTCam.h"
 #import "FLTSavePhotoDelegate.h"
 
+@interface FLTImageStreamHandler : NSObject <FlutterStreamHandler>
+
+/// The queue on which `eventSink` property should be accessed
+@property(nonatomic, strong) dispatch_queue_t captureSessionQueue;
+
+/// `eventSink` property should be accessed on `captureSessionQueue`.
+/// The block itself should be invoked on the main queue.
```

Comment:
A declaration comment should start wit a description of what something is, before getting into how to use it. So:

```
/// The event sink to stream camera events to Dart.
///
/// The property should only be accessed on `captureSessionQueue`.
/// The block itself should be invoked on the main queue.
```